components/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@brightlayer-ui/react-native-components",
-    "version": "8.0.3",
+    "version": "8.0.3-beta.1",
```

8.03 has already been released, this should be 8.0.4-beta
```suggestion
    "version": "8.0.4-beta.1",
```